pkg/descheduler/descheduler_test.go

```diff
@@ -44,20 +44,16 @@ func TestTaintsUpdated(t *testing.T) {
 	}
 	rs.Client = client
 	rs.DeschedulingInterval = 100 * time.Millisecond
-	errChan := make(chan error, 1)
-	defer close(errChan)
+
 	go func() {
 		err := RunDeschedulerStrategies(ctx, rs, dp, "v1beta1", stopChannel)
-		errChan <- err
```

Comment:
I don't think we should remove this pattern entirely (from when it was added in https://github.com/kubernetes-sigs/descheduler/pull/568)

Rather, I think the panic you're seeing could be fixed by moving the nil check to here from L55, so we're not always sending nil over this channel. wdyt?